src/services/email/serializer.ts

```diff
@@ -44,7 +44,7 @@ export const generateTicketsEmail = (cart: DetailedCart) =>
       highlight: "Bienvenue à l'UTT Arena&nbsp;🔥&nbsp;!",
     },
     reason:
-      "Tu as reçu cet email car tu es inscrit à l'UTT Arena 2021. Si ce n'est pas le cas, contacte-nous et change le mot de passe de ta boîte mail.",
+      "Tu as reçu cet email car tu es inscrit à l'UTT Arena 2022. Si ce n'est pas le cas, contacte-nous et change le mot de passe de ta boîte mail.",
```

Comment:
Si tu peux modifier comme l'indique la com 😉 
```suggestion
      "Tu reçois ce mail car tu es inscrit à l'UTT Arena 2022. Si ce n'est pas le cas, contacte-nous et change le mot de passe de ta boîte mail.",
```